src/readme.txt

```diff
@@ -2,9 +2,9 @@
 Contributors: newsletter2go
 Donate link: http://www.newsletter2go.com
 Tags: email, email marketing, emailing, email marketing software, newsletter, newsletter software, newsletter plugin, newsletter widget, content, content transfer, subscribers
-Requires at least: 3.0.1
-Tested up to: 3.4
-Stable tag: trunk
+Requires PHP: 5.2.4
```

Comment:
Maybe it should be something like `Requires PHP: 5.2.4 or higher`

Reply:
It will be shown like this on our plugin-page: 
![image](https://user-images.githubusercontent.com/6818297/51825106-59715c00-22e4-11e9-9b2b-a1d9cb0ead04.png)